remotes/remotes.go

```diff
@@ -167,7 +170,8 @@ const (
 	// https://en.wikipedia.org/wiki/Exponential_smoothing#Basic_exponential_smoothing
 	// for details.
 	remoteWeightSmoothingFactor = 0.5
```

Comment:
not sure agreeing with this, in case of connectivity issues a failure to this moment IMO has more weight than the previous history, I would suggest to move it to 0.8